src/backend/api/meals.js

```diff
@@ -2,11 +2,96 @@ const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
+
+// GET api/meals/ query parameters
+
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    if (request.query.maxPrice) {
+      const maxPrice = parseInt(request.query.maxPrice);
+      const mealsWithMaxPrice = await knex("meals").where("price", "<", maxPrice);
+      response.json(mealsWithMaxPrice);
+    }
+    else if (request.query.tittle) {
```

Comment:
This query parameter is called `title`, not `tittle`. The DB column should also be `title`.